### PR TITLE
review Chapter 8

### LINKED ISSUE #35: Review Ch 8
Review of chapter 8 is needed.

### PR DESCRIPTION
**Pull request by IttayWeiss**
Finished the review. Corrected some typos, and some other minor things. Please compare carefully to see if you agree with all of the changes. 

Closes #35

### CODE REVIEW

File: src/ch08-02-strings.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ fn add(self, s: &str) -> String {
 {{#rustdoc_include ../listings/ch08-common-collections/listing-08-14/src/main.rs:spanish}}
 ```
 
-במקרה זה, ערכו של `len` יהיה 4, מה שאומר שהווקטור המאחסן את המחרוזת "Hola" הוא באורך 4 בתים. בקידוד UTF-8, כל אחת מהאותיות הללו דורשת 1 בייט. השורה הבאה, לעומת זאת, עשויה להפתיע אותך. (שים לב שהמחרוזת הזו מתחילה באות הקירילית הגדולה Ze, לא המספר הערבי 3.)
+במקרה זה, ערכו של `len` יהיה 4, מה שאומר שהוקטור המאחסן את המחרוזת "Hola" הוא באורך 4 בתים. בקידוד UTF-8, כל אחת מהאותיות הללו דורשת 1 בייט. השורה הבאה, לעומת זאת, עשויה להפתיע אותך. (שים לב שהמחרוזת הזו מתחילה באות הקירילית הגדולה Ze, לא בספרה הערבית 3.)
```
Comment (IdanLib) [inline src/ch08-02-strings.md:150]: Oh snap, two totally different things. Thanks for the correction.

**Review by IdanLib — COMMENTED** [review]
- Corrections for typos etc and formatting errors - look good to me! Same for the מספר vs ספרה correction.
- Loved ערכים התחלתיים instead of ערכים ראשוניים. We should make the change throughout the chapter. 
- Double vs single "vav" - see general comment with link. 

Let me know how you would prefer to handle the review and PR. Should I request changes?

**Comment by IttayWeiss** [comment]
All makes sense, and good points. I suggest to proceed as follows. Anything that can be differed to an issue, should be differed to an issue (e.g., a consistent use of how many vavs to use in the words vector and variant). Anything else, if any, you can comment on as a request to change. So, if that's ok with you, if you can just decide per issue above, if you're going to put it as an issue, or a request to change, and then I'll respond to requests, and merge the PR.

**Comment by IdanLib** [comment]
OK, reviewing my comments again - it's essentially down to the double "vav" and the ראשוניים vs התחלתיים. So, let's keep it simple - merge the PR, and in the meantime I'll open two issues, one for the double vav, the other for ראשוניים vs התחלתיים. Once settled, I'd like to make the  double vav issue (when and where) part of the editing guidelines for future contributors, to ensure consistency.

**Pull request merged**